docs/internals/howto-release-django.txt

```diff
@@ -458,16 +551,17 @@ Now you're ready to actually put the release out there. To do this:
    (If this is a security release, what follows should be done 15 minutes
    before the announced release time, no sooner.)
 
+#. If this is the alpha release of a new series, before you can upload the
+   alpha release package to the djangoproject server, you will need to
+   **first** create the directory A.B.
```

Comment:
I got burnt by this. I know it's documented, I think maybe our release script might want to warn about this and ideally give a command to create a new directory :thinking: 
As most of the instructions are automated by the script I didn't catch this :disappointed: